lib/Connection.php

```diff
@@ -314,8 +314,8 @@ public function query($sql, &$values=array())
 	{
 		if ($this->logging)
 		{
-			$this->logger->log($sql);
-			if ( $values ) $this->logger->log($values);
+			$this->logger->debug($sql);
+			if ( $values ) $this->logger->debug("Values:", $values);
```

Comment:
This change isn't enough. [The `Config` class does reflection to verify that the passed instance actually implements a public `log` method.](https://github.com/jpfuentes2/php-activerecord/blob/891ff0d5d4f5423583a31538f48fddd75321c577/lib/Config.php#L247-L248)